konveyor/tackle-ui
# Add daily e2e executions

## What is included in this PR

This PR adds a new GH Action workflow `ci-daily` which is going to execute the e2e tests:
- Daily at 00:00
- When any developer wants to fire it since it includes [manual triggering](https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/) of workflows.

The reason why we need a continuous check of all backends and the UI is because the backend microservices can change at any point in time and the UI has no way of knowing whether or not that change affects the UI or not.

- This PR is going to test all the latest container images of the tackle project.

## How to test this PR
Since the new GH Action will work only in the `main` branch you need to checkout this PR and then set all the changes of this PR into your own `main` branch. You should see this PR in action going to the "Actions" tab of your Github forked repository.
